translations/fr.json

```diff
@@ -353,12 +344,11 @@
   "actionbox.unit.toomany.info": "Cette ville ne peut plus contenir d'unités. Améliorez votre ville pour avoir plus d'espace pour les unités.",
   "actionbox.building.doit": "LE FAIRE",
   "actionbox.building.requiredtech": "Vous avez besoin de rechercher {0} pour faire ça, cliquez sur le bouton 'Recherche' .",
-  "actionbox.building.techtree": "RECHERCHES",
+  "actionbox.building.techtree": "ARBRE DE RECHERCHES",
```

Comment:
Yes, the translation is true, and it was the first translation that i used. But in game, the term is too big and produce superposition betweend buttons.